app/Resources/views/events/show.html.twig

```diff
@@ -61,9 +67,8 @@
                 <div class="text-center event-wiki-stats--empty">
                     {% if event.hasJob %}
                         &nbsp;
-                    {% elseif event.numParticipants == 0 %}
-                        {# FIXME: ^ also check for event.numCategories == 0 and add msg('categories') once T194707#4620358 is resolved #}
-                        {{ msg('event-misconfig', [msg('participants')]) }}
+                    {% elseif event.numParticipants == 0 and event.numCategories == 0 %}
+                        {{ msg('event-misconfig', [[msg('participants'), msg('categories')]|join(', ')]) }}
```

Comment:
Not that it matters for this commit, but in https://github.com/wikimedia/ToolforgeBundle/pull/13 I'm adding a `list_format` Twig function that'll help with lists like this.

Reply:
Good to know! I think this line of code will get removed with #234 anyway.